Modix.Bot/Modules/AuthorizationModule.cs

```diff
@@ -17,85 +18,84 @@ namespace Modix.Modules
     [HelpTags("claims")]
     public class AuthorizationModule : ModuleBase
     {
+        private readonly IAuthorizationService _authorizationService;
```

Comment:
Private members should come at the end of a class.

Reply:
Huh, never seen this approach before. Everywhere I've worked we've always stuck them at the top, because scope (ime) is top-down. Does anyone else have an opinion on this?

Reply:
It's a pretty subjective thing. At my workplace, we put them at the bottom as well.

Reply:
Always put them at the top. Never at the bottom

Reply:
We don't really have a convention for this in the codebase - some code does it one way, some the other; personally I also agree that fields/properties regardless of protection should be at the top.

Reply:
Well, we can have a philosophical discussion in more detail somewhere else, but for me it comes down to "most likely for a random person to want to look at" being closer to the top. `private` values definitely don't fall in that category.

Leave it as is for now, I guess.